packages/test/test-end-to-end-tests/src/test/benchmark/DocumentMultipleDds.ts

```diff
@@ -0,0 +1,241 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import {
+	ContainerRuntime,
+	IContainerRuntimeOptions,
+	ISummarizer,
+} from "@fluidframework/container-runtime";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import {
+	ContainerRuntimeFactoryWithDefaultDataStore,
+	DataObject,
+	DataObjectFactory,
+} from "@fluidframework/aqueduct";
+import { SharedMap } from "@fluidframework/map";
+import { SharedString } from "@fluidframework/sequence";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { IContainer } from "@fluidframework/container-definitions";
+import {
+	createAndAttachContainer,
+	createLoader,
+	createSummarizerFromFactory,
+	summarizeNow,
+} from "@fluidframework/test-utils";
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { IDocumentLoaderAndSummarizer, IDocumentProps, ISummarizeResult } from "./DocumentCreator";
+
+class TestDataObject extends DataObject {
```

Comment:
Ideally, tests use the default data object that the test object provider provides. However, it only creates1 DDS and in these tests we want to create multiple (3) DDSes per data store.
Can you add a comment to reflect that?